ox_profile/core/metrics.py

```diff
@@ -51,11 +51,9 @@ def snap(self, frame):
 
         """
         stack = []
-        self.name = '{}({})'.format(frame.f_code.co_name,
-                                    frame.f_globals.get('__name__'))
+        self.name = '%s(%s)' % (frame.f_code.co_name, frame.f_globals.get('__name__'))
```

Comment:
I'm OK with changing string formatting but would like to keep to the convention that source code lines are less than 80 characters. Please fix formatting to match that python convention.

Reply:
Ok. I made those lines less than 80 characters length again. My intention to get rid of the .format() was that it is a bit slower than %. (as far as I know)